src/locales/de/config.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
This file does not seem to be translated

Reply:
No the issue is that I put the German translation into the wrong file .. swapped the files in `en/` and `de/` now everything is good ;)

Reply:
if you opt for a literal translation you would stick to Hobby. model trains would be a rather interesting specific example but not a translation :D if one would want a German wordone could also go for Freizeitbeschäftigung but no one uses that except people who like to avoid any kind of anglicism